dev-php/pecl-rdkafka/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>oleksandr.trotsenko@gmail.com</email>
```

Comment:
You probably need to add the proxy team as well if you're not a developer.

Reply:
Thank you sir! I hope now it's taken care of. :)